RELEASE-NOTES.txt

```diff
@@ -4,6 +4,8 @@
 -----
 - [***] Push notifications are now supported for all connected stores. [https://github.com/woocommerce/woocommerce-ios/pull/5299]
 - [*] Fix: in Settings > Switch Store, tapping "Dismiss" after selecting a different store does not switch stores anymore. [https://github.com/woocommerce/woocommerce-ios/pull/5359]
+- [*] Products: Variation number added in variation list and details page
+    [https://github.com/woocommerce/woocommerce-ios/issues/4846]
```

Comment:
Hello @anas-p 👋 
Please replace this URL with the PR's URL.

Reply:
Hi @selanthiraiyan I will change this and send a new PR with other changes.